idea_town/urls.py

```diff
@@ -17,6 +19,7 @@
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
     url(r'^api/', include(router.urls)),
     url(r'^api/metrics/', metrics_view.MetricsView.as_view(), name='metrics'),
+    url(r'^api/users/self', users_views.SelfView.as_view(), name='users_self'),
```

Comment:
I figured no, since we just removed trailing slashes from other APIs. I usually want to see slashes on things that give collections and no slash for single items